
This patch file includes the changes required to add
compat-wireless support. Besides all the patches in this
directory we have our own compat-2.6.*.[ch] files which
help backport more. These the respective compat-2.6.2x.c
files get compiled and enabled if you are on older kernels
to bring in support for new features when possible.

Lastly, the cfg80211.h change is what allows us to
get all compat-2.6.*.h files included. Its our only
point of entry through compat code. It works because
all drivers we are supporting through there one way
or another depend on cfg80211. New wireless drivers
are either mac80211 drivers or cfg80211 drivers, and
all mac80211 drivesr are cfg80211 drivers.

If your driver is not a cfg80211 driver you must
ensure you include compat.h then. I think we
added compat.h also for ssb, I forget.

--- a/include/net/cfg80211.h
+++ b/include/net/cfg80211.h
@@ -23,6 +23,7 @@
 /* remove once we remove the wext stuff */
 #include <net/iw_handler.h>
 #include <linux/wireless.h>
+#include <linux/compat-2.6.h>
 
 
 /**
